pom.xml

```diff
@@ -71,6 +71,15 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <includes>
+                    <include>version.properties</include>
```

Comment:
A very similar one. A pom.properties (not .xml!) with the same "variables" substituted in the building process.

https://github.com/telefonicaid/fiware-connectors/blob/master/flume/src/main/resources/pom.properties#L5